src/book/03-guides/03-hardware/03-batteries.mdx

```diff
@@ -0,0 +1,101 @@
+
+1. Plug the battery cable with the white plug into the corresponding socket on the side of the charger. The plug should only fit in one orientation so check before
+   plugging it in. If you find yourself forcing the plug into the socket you are doing something wrong.
+
+2. Plug the other battery cable into the main output cable of the charger, this may require an adaptor, as seen in the section on using in the robot
+
+3. Plug the Powertech Plus into a power socket, and use the arrow buttons to select the "Charging" setting for the type of battery you are charging. For the NUgus robot
+   battery this is a 14.8V 4 Cell, as seen in the image below. For a Darwin robot battery, this is a 11.1V 3 Cell.
+
+4. Hold down the start button until the device beeps, and observe the content on the screen. If the charging device indicates any errors, stop charging immediately,
+   unplug the battery, and return it to a fire retardant case. If there are no errors, press start to commence charging
+
+5. Close and lock the white BAT-SAFE box with the battery inside (Keep the charger outside if possible to hear when it is done)
+
+6. Stay in the lab with the battery and disconnect immediately when charge has completed. Never leave a battery charging when there is no one in the lab
+
+![Charging setting on Powertech Plus LiPo Charger](./images/batteries/charge.jpg)
+
+## Storing and Traveling with batteries
+
+<Alert>
+
+Care should be taken to verify the LiPo battery limits of the airline you are travelling with. They will usually specify a maximum `Ah` value for all batteries and
+possibly a maximum number of batteries. If you are found carrying more than this limit then you will likely need to discard the batteries and not fly with them.
+</Alert>
+
+When travelling, LiPo batteries should always be kept in fire retardant bags, with one battery stored in each fire retardant bag. If travelling on an aircraft,
+LiPo batteries should always go in carry-on luggage, never in checked luggage. It is important to prepare batteries for storage when travelling on an aircraft.
+Batteries should also be prepared for storage when being stored in the lab for an extended period of time.
+
+In both cases, preparing a battery for storage is done by following the same process outlined for charging, however by using the "Storage" setting, as seen in the
+image below.
+
+![Storage setting on Powertech Plus LiPo Charger](./images/batteries/storage.jpg)
+
+## Disposal of batteries
+
+If a battery has been physically damaged or is showing errors when attempting to charge, it should be disposed of. Batteries for disposal should be taken in a fire
+retardant bag to the Electrical Engineering building on campus. Talk to the team leader to confirm the most up to date details of where to take them.
+
+If the cables on the battery are damaged, it is a good idea to wrap the damaged section in electrical tape to prevent them short cicruiting inside the fire retardant
```

Comment:
Specify that the single cable must be wrapped, it could be interpreted that the entire section needs to be wrapped together with this wording (which would obvs cause the short then)